Charo aporte4

charo007 opened the pull request:
<!-- Saludos, si usted esta leyendo esto, significa que quiere colaborar con el proyecto aportando quizás mejoras o información que encontró mientras utilizo el mismo, es por eso que queremos agradecerle que se tome su tiempo por hacer lo antes mencionado. -->

<!-- 
	Utilice la siguiente forma de títulos-
•	Si es una mejora en la base de datos [DB] titulo
•	Si es una mejora de estilos [CSS] titulo
•	Si es una mejora en el código [HTML] titulo
Si utiliza varias mejoras, no especifique nada, solo el titulo.

--> HTML, CSS y archivos de imágenes.

<!-- Necesitamos el que titulo del tema sea lo mas representativo posible. -->

##### ¿QUÉ CAMBIO PROPONE?:
<!--
	Agregue un pequeño texto que explique cual es el bug que desea reportar o cual es la situación de mejora que le gustaría que apliquemos. 
-->

Agregué una sección para la galería del sitio, en la cual el usuario puede ver los trabajos que se han realizado en el taller. Esta galería tiene un archivo .html y archivo de estilos llamado galeria.css.
Además, agregué una carpeta con las imágenes que utilicé para dicha galería.

##### NOTAS EXTRA:
<!--
Cualquier información que pueda ayudar a los desarrolladores a identificar y solucionar el problema debe ser puesta aquí.
-->

##### BRANCH(ES):
<!-- 
Especifique la(s) rama(s) afectada(s) por este problema: master, 0.x, 1.x, u otra rama. 
Reemplace el valor de master por la rama correspondiente.
-->

master



##### OTRAS PERSONALIZACIONES:
<!-- 
En caso de que se produzca alguna mejora en el código, avísennos, caso contrario, no es necesario que usted escriba nada aquí. (solo para programadores)
-->




<!-- ------------------------- THE END ------------------------------
 Gracias nuevamente por su colaboración, atenderemos su tema lo antes posible. 
-->

charo007 committed:
Modifiqué el archivo index

Hice modificaciones al archivo index, estas modificaciones se
basaron en agregar un carrusel de imágenes para hacer que el sitio
sea más atractivo para el usuario.

charo007 committed:
Agregando el archivo de estilos para la sección galeria

En este archivo apliqué los estilos para darle diseño a la sección
galeria.

charo007 committed:
Agregando la galeria del sitio

En este archivo .html, agregué la galeria con imágenes para el sitio.

charo007 committed:
Archivos para la galeria

Estas son las imágenes que utilicé para crear la galeria.

RainTrap reviewed (changes requested):
Dejar una linea en blanco al final de los archivos galeria.css y galeria.html para evitar futuros inconvenientes.

charo007 committed:
Agregando los espacios solicitados al final de los archivos.

Agregué el espacio al final del código en los archivos que solisitaste.
Además, también lo agregué en otro de los archivos que tenía
ya que el mismo no estaba.

RainTrap reviewed (approved):
Gracias por realizar los cambios solicitados. El código esta correcto.
